emails/views.py

```diff
@@ -61,27 +74,26 @@ def inbound(request):
         return HttpResponse("Address does not exist")
 
     # Store in local DB
-    message = Message.objects.create(
+    db_message = Message.objects.create(
         relay_address=relay_address,
         from_address=from_address,
         subject=subject,
         message=text
     )
 
     # Forward to real email address
-    try:
-        message = Mail(
-            from_email='inbound@privaterelay.groovecoder.com',
-            to_emails=relay_address.user.email,
-            subject='Forwarding email from %s sent to %s' % (
-                from_address, local_portion
-            ),
-            html_content=text
-        )
-        sendgrid_client = SendGridAPIClient(config('SENDGRID_API_KEY'))
-        response = sendgrid_client.send(message)
-        print(response)
-    except Exception as e:
-        print(e.message)
-
-    return HttpResponse(message)
+    sl_message = BasicMessage()
+    sl_message.subject = 'Forwarding email from %s sent to %s' % (
+        from_address, local_portion
+    )
+    sl_message.html_body = html
+    sl_message.plain_text_body = text
+    sl_message.from_email_address = EmailAddress(
+        'inbound@privaterelay.groovecoder.com'
```

Comment:
Should we make this configurable in the PR?

Reply:
Good catch, yes!